docs/en/observability/whats-new.asciidoc

```diff
@@ -11,4 +11,276 @@ https://www.elastic.co/blog/elastic-observability-7-6-0-released[7.6] |
+that take bursty or seasonal traffic patterns into account and notify a user only when true anomalies are detected.
+
+[role="screenshot"]
+image::images/anomaly-detect-alert.png[Anomaly detection alerts]
+
+[discrete]
+== APM PHP agent (beta)
+
+We are excited to introduce the PHP agent as a beta to Elastic APM. With this new agent, you get instant visibility
+into the performance of your PHP applications. The addition of the PHP agent now extends our support of all the major
+application development languages, which includes Java, .NET, GO, Python, Ruby, and Javascript.
+
+[discrete]
+== APM Java agent: Micrometer integration
+
+With the Java agent release 1.18.0, we have added support for automatic metrics collection from Micrometer. The
+metrics include cache statistics, thread pools, page hits, and also your custom business-related metrics, such as
+revenue which you can view in {kib}. For information on how to set up and start collecting these
+metrics, see our {apm-java-ref}/metrics.html#metrics-micrometer[Micrometer documentation].
+
+[discrete]
+== OpenTelemetry metrics intake
+
+We extended the https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/master/exporter/elasticexporter[Elastic exporter]
+for OpenTelemetry Collector to collect metrics and export them to the APM server. The Elastic OpenTelemetry exporter
+now supports both traces and metrics collection. OpenTelemetry metrics help to monitor business KPIs and technical
+components that are not auto instrumented by the OpenTelemetry SDK.
+
+Capturing insightful business metrics on applications with OpenTelemetry looks like this:
+
+[source,opentelemetry]
+----
+// initialise metric
+Meter meter = OpenTelemetry.getMeter("my-frontend-frontend");
+DoubleValueRecorder orderValueRecorder = meter.doubleValueRecorderBuilder("order").build();
+
+
+public void createOrder(HttpServletRequest request) {
+
+   // create order in the database
+   ...
+   // increment business metrics for monitoring
+   orderValueRecorder.record(orderPrice);
+}
+----
+
+[discrete]
+== Disk spooling for Beats (beta)
+
+In {minor-version}, we have included the beta release of a new disk {filebeat-ref}/configuring-internal-queue.html[queue]
+that stores the pending events on disk rather than the main memory. This enables Beats to queue a larger number of events
+than what is possible with the memory queue, and also to save events when a Beat or device restarts. It provides increased
+availability and reliability for cases like transient errors of the output; for example, {es} is not available for some time.
+
+[discrete]
+== Enhanced overview page
+
+In {minor-version}, the Observability overview page has been enhanced to include the User Experience chart.
+
+[role="screenshot"]
+image::images/obs-overview-ue.png[Enhanced overview page]
+
+[discrete]
+== Give {minor-version} a try
```

Comment:
I think we should provide this link earlier for users who won't read the whole document. WDYT?

Reply:
Good idea. I moved it to the top.